openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -11,10 +17,63 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
+from openedx_events.learning import signals  # pylint: disable=reimported
+from openedx_events.tests.utils import FreezeSignalCacheMixin
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# If a signal is explicitly not for use with the event bus, add it to this list
+KNOWN_UNSERIALIZABLE_SIGNALS = ["org.openedx.learning.discussions.configuration.changed.v1"]
+
+# TODO (EventBus): figure out how to automatically pull new signals in
```

Comment:
[question] Is this still accurate? I'm missing where we are doing this manually. Please remove if it no longer applies.

Reply:
I was under the impression that for signals that fail the tests, we document it in the known unserializable signals so the tests will always skip these and run tests on only working signals to see if old ones break after new code is put in, or newer signals break, we manually put in the signals onto the KNOWN_UNSERIALIZABLE_SIGNALS list

Reply:
I think I am on the same page about your PR comment. But, I still don't understand the purpose of this "TODO" comment in the code. Should it just be removed? If not, can you help me better understand what it means?

Reply:
@whuang1202: I see you removed this. Does that mean you discussed with Becca and agreed it was no longer relevant? I often find it useful to get PR comment responses to explain the changes.

Reply:
I took it out temporarily since it didn't make much sense, I'll put up a clearer todo message for the next commit since we're referring to importing signals